metacraft/executive-summary.md

```diff
@@ -1,15 +1,13 @@
 ### Executive Summary 
 
-“Amateurs together craft a Universe” — Metacraft
+“What does it take to create something with zero skill and knowledge of that field?”
 
-Metacraft at first is a “decentralized” Game Studio, we’re amateurs, come from various backgrounds connected at heart by curiosity, enthusiasm and the burning desire to create “interesting" stuffs.
+Lots of time, by yourself, or, lots of helps, from everyone! 
 
-Believing people who play Game know how it should be built, and on the power of crowdsourcing, like Wikipedia. Metacraft established as a platform to empower us all with the ability to collaborate with anyone, anywhere, anytime, to build a universe called `ATEM` together. Metacraft on it’s side appreciate the contributors fairly and transparently by a token called `Metacraft Mineral`.
+Triggered with that idea, Metacraft, as a Decentrialized Game Studio, is a collective, or “tribe" as how we called ourselves, of individuals come from various backgrounds but connected at heart by curiosity, enthusiasm and the burning desire to create “interesting" stuffs.
 
-We hope this could potentially change the way people create Games, at least this offer another way for amateurs like us have chance to hands-on create Games the way we love.
+To enable our vision, we have our own proposal called [Metacraft Framework]. It is a complex process to bring to life a simple goal — Allow anyone , from anywhere, with any background, skill set, to easily join co-crafting the Metacraft Universe. By rewarding contributors with the project's token: $MEMI, we are also able to appreciated fairly and transparently our contributors community for  their contribution  toward the growth of the project.
```

Comment:
we could use Markdown syntax to attach link to places like [Metacraft Framework], refer [this document](https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses) for more detail